src/style.css

```diff
@@ -1 +1,56 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  --hover-color: #00acdc;
```

Comment:
It's better to declare the color variable at the beginning of the file (at ::root or html), or in a separate file